### PR TITLE
Format and render time

### PR DESCRIPTION
**Pull request by stulees**
## Description
When configured for `time` or `timedate` types, formatting the current time according to _time_ attribute and rendering the value to HTML

## Motivation and Context
Component development

## How Has This Been Tested?
Tested locally via experimenting with demo. Also tested with example template using staged version of component. 

https://apps-stage-8.risevision.com/templates/edit/b38bc817-6acf-4023-a7b1-9b7204838b5f/?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
n/a

### CODE REVIEW

File: src/rise-time-date.js [hunk 1]
```diff
@@ -127,14 +138,36 @@ export default class RiseTimeDate extends RiseElement {
...
+  }
+
   _sendTimeDateEvent( eventName, detail ) {
     super._sendEvent( eventName, detail );
 
     // TODO: handle setting uptime
   }
 
+  _runTimer() {
+    this._refreshDebounceJob = Debouncer.debounce( this._refreshDebounceJob, timeOut.after( 1000 ), () => this._processTimeDate() );
```
Comment (santiagonoguez) [inline src/rise-time-date.js]: @stulees I'm not familiar with the debouncer concept ? Is this similar to a setTimeout, or what does it provide ?

File: src/rise-time-date.js [hunk 2]
```diff
@@ -1,14 +1,16 @@
 /* eslint-disable no-console */
 
 import { html } from "@polymer/polymer";
+import { timeOut } from "@polymer/polymer/lib/utils/async.js";
+import { Debouncer } from "@polymer/polymer/lib/utils/debounce.js";
```
Comment (santiagonoguez) [inline src/rise-time-date.js]: @stulees does these new imports impose requirements to the example HTML header imports ?

File: src/rise-time-date.js (hunk 1 above)
Comment (stulees) [reply src/rise-time-date.js]: @santiagonoguez The advantages of it are a bit wasted in my usage here, but essentially debouncer collapses multiple callbacks into one invocation after a timer, as well as ensures a _microtask_ (i.e. `_processTimeDate()`) is only called once despite if `_runTimer()` is called multiple times within the same microtask. 

I could change this to just use the `timeOut` which is essentially a `setTimeout` if you think this should be simplified. I've used Debouncer here cause we've consistently used it in other components.

File: src/rise-time-date.js (hunk 2 above)
Comment (stulees) [reply src/rise-time-date.js]: @santiagonoguez These imports have been imported in previous components and have not required templates to add to their header imports.  It's not clear to me why you think they would have to. Can you clarify?

File: src/rise-time-date.js (hunk 1 above)
Comment (santiagonoguez) [reply src/rise-time-date.js]: @stulees I don't think we should use setTimeout if this is more robust and we can trust it, but I was just curious, thanks !

File: src/rise-time-date.js (hunk 2 above)
Comment (santiagonoguez) [reply src/rise-time-date.js]: @stulees if those have worked ok with other components, then I agree with you it's safe, thanks !

**Comment by stulees** [comment]
@santiagonoguez I decided to simplify and use `timeOut` only instead of `Debouncer`. It's one less import, avoiding unnecessary bloat and makes the code more clear. I don't foresee this component requiring `Debouncer`, but if need be we can always refactor to use it again.

**Pull request merged**